server/models/Availability.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
```

Comment:
If you didn't use this, then consider removing it to keep your code clean.

Reply:
OK